content/600-about/200-prisma-docs/20-style-guide/02-word-choice.mdx

```diff
@@ -213,17 +213,28 @@ Open a terminal window.
 
 This can be a noun ("setup") or verb ("set up") and as such can cause confusion. Try to avoid, and use an alternative term. For example, "configure" or "enable".
 
-If you must use it, ensure you use the correct form. Remember to check in code snippets, where it might lurk in the code comments.
+If you must use it, ensure that you use the correct form. Remember to check in code snippets, where it might lurk in the code comments.
 
 ### Relations
 
 When referring to relations, use the following forms:
 
-| Avoid                                           | Good                                      |
-| ----------------------------------------------- | ----------------------------------------- |
-| 1-1                                             | One-to-one                                |
-| 1-n                                             | One-to-many                               |
-| m-n                                             | Many-to-many                              |
+| Avoid | Good         |
+| ----- | ------------ |
+| 1-1   | One-to-one   |
+| 1-n   | One-to-many  |
+| m-n   | Many-to-many |
+
+### The `prisma` root command
+
+We often refer to `prisma` CLI commands such as `prisma db push`. The short form of these commands omits the `prisma` root command. For example: `db push`.
+
+`prisma` CLI commands always work with `prisma` included, but they do not always work without it. Therefore, in most cases we should include the root command to help docs users who drop into the docs at that point. Use the following guidelines when you write about this command:
```

Comment:
They actually _never_ work without the mention of the executable of the CLI, `prisma`. We just talk about them in context of knowing that this is all under `prisma` sometimes (e.g. CLI reference) so that is helpful to keep things shorter.

Reply:
Thanks @janpio , I'll update the doc to reflect this